platform/about-these-features/multi-node.md

```diff
@@ -30,32 +30,42 @@ Starting a multi node run in CLI is the same as how you run any other script, Gr
 
 For example --gpus 5 --instance\_type 4\_v100\_32gb will automatically trigger multi node training
 
-Example of 1 node 4 GPUs :
+Prior to being able to run the below you will need to clone a fork of the Pytorch-lightning repo (https://github.com/filintod/pytorch-lightning). For convenience the code to do so is below.
 
-```bash
-grid run --instance_type 4_v100_16gb --framework lightning --gpus 4 torch_elastic_vae.py --data_path grid:stl10:1
+```
+git clone https://github.com/PyTorchLightning/pytorch-lightning.git
+cd pytorch_lightning
+printf "torchvision==0.11.1 \njsonargparse==4.1.0" >> requirements.txt
 ```
 
-Example of 2 nodes 8 GPUs:
+Example of 1 node 1 GPU :
 
 ```bash
-grid run --instance_type 4_v100_16gb --framework lightning --gpus 8 torch_elastic_vae.py --data_path grid:stl10:1
+grid run \
+--name single-node-test \
+--strategy grid_search \
+--instance_type g4dn.xlarge \
+--framework lightning \
+--gpus 1 \
+pl_examples/basic_examples/mnist_examples/image_classifier_5_lightning_datamodule.py \
+--trainer.gpus 1 \
+--trainer.strategy 'ddp' \
+--trainer.max_epochs 50
 ```
 
-## Step 0: Start a Run using Web UI
-
-Start a new run
-
-Paste the Github script in Run dialog as you would normally
-
-Select the machine and specify GPUs per experiment
+Example of 4 nodes 4 GPUs:
 
-![](https://lh5.googleusercontent.com/SFa4W\_pTQZsnAevsho\_gX8VxoH9qtKfcoTB-8hsR6QECZw-Q1R7TT0ecGMYTvwIwZN7Z1kFNd7OsAI1HmreacwMLOi994u74buj0BX6GOZfeqZ59yLmQ2pGIMJPqSJJACjTpa-Ny)
-
-## Step 1: Visualize Results
-
-In the same way as you would for a single node run. If tensorboard logs are available, scalars will be visible and artifacts such as checkpoints and configurations can be downloaded
-
-Verify how many GPUs were run in the Experiment Details
+```bash
+grid run \
+--name 4-node-test \
+--strategy grid_search \
+--instance_type g4dn.xlarge \
+--framework lightning \
+--gpus 4 \
+pl_examples/basic_examples/mnist_examples/image_classifier_5_lightning_datamodule.py \
+--trainer.gpus 4 \
+--trainer.strategy 'ddp' \
+--trainer.max_epochs 50
+```
```

Comment:
double check this is correct. in order for 4 nodes to launch, instance should have 1 gpu on it, since we're passing --gpus 4